### DISCUSSION TITLE
Static files for Bootstrap5

### DISCUSSION THREAD

**Discussion by GilShalit** [Megabit/Blazorise#5162] [Q&A]
The quickstart documentation is geared towards the Bootstrap4 provider. What are the correct and <script> tags to use for bootstrap.css and bootstrap.js, respectively?

I am on version 1.3.3.

Thanks!

**Comment by stsrki** [comment]
All other providers are listed on https://blazorise.com/docs/usage

BS5 guide is on https://blazorise.com/docs/usage/bootstrap5

**Comment by David-Moreira** [comment]
Why do you say it's geared towards bootstrap 4?
Theres docs for each provider. Here's bs5:
https://blazorise.com/docs/usage/bootstrap5

**Reply by GilShalit** [reply]
My bad, I missed the navigation to the dif providers. Thanks!

Closing

**Comment by GilShalit** [comment]
The information is where it should be!
